### PR TITLE
Add/update empty metadata instance list message

### PR DESCRIPTION
**Pull request by ptitzler**
Display standardized  'getting started'  message in the metadata editor list if no instances are defined:

image
The highlighted text is replaced with the schemaspace display name.

### What changes were proposed in this pull request?

- Added/updated message that is displayed when no metadata instance is defined for all currently supported schemaspaces

### How was this pull request tested?

Deleted all metadata instances and confirmed that the message text is displayed as expected

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by ajbozarth — CHANGES_REQUESTED** [review]
I think adding this to the generic `MetadataWidget` makes sense, but since `renderDisplay` is an overridable function, this checkout would need to be reimplemented for each instance (as it is in code snippets and runtimes). As such I dont think we need to make those generic.

**Review by ajbozarth — APPROVED** [review]
Per a slack convo with Patrick I'm going to say this code LGTM, I did not test locally though. Also we will need to add this new code block to https://github.com/elyra-ai/elyra/pull/2532 when we take that back out of draft

File: packages/code-snippet/src/CodeSnippetWidget.tsx [hunk 1]
```diff
@@ -591,7 +591,7 @@ export class CodeSnippetWidget extends MetadataWidget {
         <div>
           <br />
           <h6 className="elyra-no-metadata-msg">
-            Click the + button to add a new Code Snippet
+            Click the + button to add {this.props.display_name}
```
[the remark this answers is not in the document]
Comment (ptitzler) [reply packages/code-snippet/src/CodeSnippetWidget.tsx:594]: You are right in that the rendered result is the same! However, if we ever were to change the display name for this schemaspace, then we'd have to update the hard coded string.

**Comment by ptitzler** [comment]
Would be open to lowercasing the display name in the message to make it look less awkward

**Pull request merged**